x/ccv/provider/types/keys.go

```diff
@@ -106,7 +106,7 @@ const (
 	// ThrottledPacketDataBytePrefix is the byte prefix storing throttled packet data
 	ThrottledPacketDataBytePrefix
 
-	// PendingSlashPacketEntryBytePrefix is the byte prefix storing global slash queue entries
+	// GlobalSlashEntryBytePrefix is the byte prefix storing global slash queue entries
```

Comment:
Using this change to point out that the `SlashMeterByteKey` and  `LastSlashMeterFullTimeByteKey` constants above should have their name and doc updated respectively to sthg like:
```
// SlashMeterBytePrefix ...
-SlashMeterByteKey
+SlashMeterBytePrefix

----

-// LastSlashMeterReplenishTimeBytePrefix ...
+ // LastSlashMeterFullTimeByteKey ...
LastSlashMeterFullTimeByteKey
```

Reply:
The comments were off, thanks!